### PR TITLE
Change url generation to dowload torrents for Torrent9.

### LINKED ISSUE #4716: BTFailure from torrent9 when trying to download torrents.
Hi, 
Yesterday, I was trying to check why my torrents were not downloading anymore from torrent9.
In fact it's due to torrent9 that change the url to get their torrents.
The `http://torrent9.red/get_torrent/blabla.html` is not working anymore and redirect to a page with non related torrent.

I have a made a quick fix in my fork: https://github.com/czotti/SickRage/tree/fix/torrent9
The fix is in the file `sickbread/providers/torrent9.py`
On my side the fix seems to work, the torrent is added to my transmission downloader.

It's my first time trying to contribute, can I make a PR with my fix ?

Branch/Commit:
OS: Freenas 11.1, Archlinux
Browser: Firefox.
What you did: Searching for torrent on torrent9 tracker.
What happened: Nothing is download due to a change in the torrent9 torrent download.
What you expected: The torrent to e downloaded.
Logs:
```
AA BTFailure: not a valid bencoded string
AA raise bencode.BTL.BTFailure("not a valid bencoded string")
AA File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/sickbeard/helpers.py", line 1935, in bdecode
AA torrent_bdecode = helpers.bdecode(result.content, True)
AA File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/sickbeard/clients/generic.py", line 216, in _get_torrent_hash
AA result = self._get_torrent_hash(result)
AA File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/sickbeard/clients/generic.py", line 259, in sendTORRENT
AA Traceback (most recent call last):
2018-05-20 09:30:20 ERROR    SEARCHQUEUE-DAILY-SEARCH :: [5a9a634] Transmission: Failed Sending Torrent
AA BTFailure: not a valid bencoded string
AA raise bencode.BTL.BTFailure("not a valid bencoded string")
AA File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/sickbeard/helpers.py", line 1935, in bdecode
AA torrent_bdecode = helpers.bdecode(result.content, True)
AA File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/sickbeard/clients/generic.py", line 216, in _get_torrent_hash
AA Traceback (most recent call last):
2018-05-20 09:30:20 ERROR    SEARCHQUEUE-DAILY-SEARCH :: [5a9a634] Unable to bdecode torrent
2018-05-20 09:30:20 INFO     SEARCHQUEUE-DAILY-SEARCH :: Downloading iZombie.S04E11.VOSTFR.HDTV.x264-Torrent9 from Torrent9
2018-05-20 09:30:20 INFO     SEARCHQUEUE-DAILY-SEARCH :: [Torrent9] :: Quality of iZombie.S04E11.VOSTFR.HDTV.x264-Torrent9 is SDTV
2018-05-20 09:30:20 INFO     SEARCHQUEUE-DAILY-SEARCH :: [Torrent9] :: Found result iZombie.S04E11.VOSTFR.HDTV.x264-Torrent9 at http://www.torrent9.red/get_torrent/izombie-s04e11-vostfr-hdtv.torrent
```

### PR DESCRIPTION
**Pull request by czotti**
Fixes #4716

Proposed changes in this pull request:
- change the torrent9 provider to support their new url to download torrent files.

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/SickRage/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: sickbeard/providers/torrent9.py [hunk 1]
```diff
@@ -64,13 +80,20 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                     continue
 
                 with BS4Parser(data, 'html5lib') as html:
-                    torrent_rows = html.findAll('tr')
+                    torrent_table = html.find('div', {'class': 'table-responsive'})
+                    if torrent_table is not None:
```
Comment (jlejeune) [inline sickbeard/providers/torrent9.py]: if torrent_table:  # is enough

**Review by jlejeune — APPROVED** [review]

**Review by lagaisse — APPROVED** [review]

**Comment by rdehouss** [comment]
This MR works for me, thx!

**Comment by Jamyz** [comment]
Great, Thanks...

**Comment by czotti** [comment]
The failing part of appveyor can be easily removed by changing the line 60 from:
`"pip install --disable-pip-version-check --user --upgrade pip"`
to:
`"python -m pip install --disable-pip-version-check --user --upgrade pip"`

Can I do this here or in a new PR ?

**Review by fly320 — APPROVED** [review]

**Review by Jamyz — APPROVED** [review]
Works Good... Thanks to push soon.

**Review by Jamyz — CHANGES_REQUESTED** [review]
Works Good....Push it Soon I Hope.....

**Pull request merged**